src/algebra/group/conj.lean

```diff
@@ -45,6 +45,18 @@ protected lemma monoid_hom.map_is_conj (f : α →* β) {a b : α} : is_conj a b
 
 end monoid
 
+section right_cancel_monoid
+variables [right_cancel_monoid α]
+
+@[simp] lemma is_conj_one_right {a : α} : is_conj 1 a  ↔ a = 1 :=
```

Comment:
Presumably this also holds for `left_cancel_monoid`, but with a different proof. It might be nicer to only generalize as far as `cancel_monoid` to avoid this weird asymmetry